src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -5,8 +5,36 @@ ifdef::cheri_standalone_spec[]
 WARNING: This chapter will appear in the unpriv spec after the RV32I chapter.
 endif::[]
 
-This chapter describes the {cheri_base32_ext_name} and {cheri_base64_ext_name} base capability instruction sets, that extend the RV32I and RV64I (or RV32E/RV64E) integer instruction sets with CHERI.
-If the hart supports RV32E/RV64E instead of RV32I/RV64I, {cheri_base_ext_name} the restrictions from <<rv32e>> apply and 16 general purpose registers instead of 32 -- but all other {cheri_base_ext_name} changes are identical so the remainder of this chapter only refers to extending RV32I/RV64I behavior.
+This chapter describes the {cheri_base32_ext_name} and {cheri_base64_ext_name} base capability instruction sets, that extend the RV32I and RV64I instruction sets with CHERI.
+Other standard options to bases are also supported such as E (16-registers), Zfinx and endianess.
+
+=== CHERI encoding formats
+
+CHERI implementations are required to choose a capability encoding format.
+The CHERI base ISA name is extended with the capability encoding format in use, and each implementation only supports one.
```

Comment:
"supports only one", and that's going to be challenging for efficient VMs, I think?

Reply:
Supporting multiple will  need to be a future feature. Krste is not happy about supportng multiple and I don't blame him - it opens the door to nightmare scenarios like variable XLEN which RISC-V are having to back away from.

I think this is sensible for now.

Reply:
I would say there should not be anything prohibiting supporting multiple encodings. You can support I and E bases already, so there could be some implementation defined mechanism to switch. I am not suggesting this is something that needs to be possible, but prohibiting it seems unnecessary.
```suggestion
The CHERI base ISA name is extended with the capability encoding format in use.
```